opacus/grad_sample/conv.py

```diff
@@ -1,16 +1,52 @@
 #!/usr/bin/env python3
 # Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
 
-from typing import Dict, Union
+from typing import Dict, Union, Tuple
 
 import numpy as np
+import time
 import torch
 import torch.nn as nn
+import torch.nn.functional as F
 from opacus.utils.tensor_utils import unfold3d
 
 from .utils import register_grad_sampler
 
 
+def unfold2d(
```

Comment:
This is probably better suited in `opacus/utils/tensor_utils.py`, at least that's where our custom `unfold3d` lives